README.md

```diff
@@ -460,6 +494,27 @@ cygnusagent.sinks.mysql-sink.mysql_password = xxxxxxxxxxxx
 # how the attributes are stored, either per row either per column (row, column)
 cygnusagent.sinks.mysql-sink.attr_persistence = column
 
+# ============================================
```

Comment:
Have you check that this configuration also works with replica sets? Typically, you have to specify the name of the replica set and consider the posibility of mongo_host of being a list of replica set elements (I can elaborate on the details if you need).